.github/workflows/formatCodeCovComment.yml

```diff
@@ -0,0 +1,24 @@
+name: Format CodeCov Comment
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  format-codecov-comment:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.issue.pull_request &&
+      contains(fromJSON('["codecov[bot]","codecov-commenter"]'), github.event.comment.user.login)
```

Comment:
Hmm, do we know for sure that that is the correct substring to be looking for? Why wouldn't it just be "codecov"? And why do we have two strings instead of just one?